etc/kayobe/inventory/group_vars/ctl-switches

```diff
@@ -0,0 +1,173 @@
+  - "switchport access vlan {{ oc_oob_vlan }}"
+
+# Interface config for workload iDRACs.
+switch_interface_config_wl_idrac:
+  - spanning-tree portfast
+  - switchport mode access
+  - "switchport access vlan {{ wl_oob_vlan }}"
+
+# Tenant VLAN range for this physical network.
+switch_interface_config_tenant_vlan_range: >-
+  {{ kolla_neutron_ml2_network_vlan_ranges |
+     selectattr('physical_network', 'equalto', sib_ctl_physical_network) |
+     map(attribute='range') |
+     join(',') |
+     replace(':', '-') }}
+
+# Provider VLAN range for this physical network.
+#switch_interface_config_provider_vlan_range: 100-199
+
+# Interface config for controllers.
+switch_interface_config_controller:
+  - spanning-tree portfast
+  - switchport mode general
+  - "switchport general allowed vlan add {{ oc_provision_vlan }} untagged"
+  # Current expect-based configuration mechanism requires no line wrapping, so
+  # VLANs must be configured individually.
+  - "switchport general allowed vlan add {{ wl_oob_vlan }} tagged"
+  - "switchport general allowed vlan add {{ wl_provision_vlan }} tagged"
+  - "switchport general allowed vlan add {{ public_vlan }} tagged"
+  - "switchport general allowed vlan add {{ internal_vlan }} tagged"
+  - "switchport general allowed vlan add {{ storage_vlan }} tagged"
+  - "switchport general allowed vlan add {{ switch_interface_config_tenant_vlan_range }} tagged"
+  # FIXME: provider VLAN range still undecided
+  #- "switchport general allowed vlan add {{ switch_interface_config_provider_vlan_range }} tagged"
+  - "switchport general pvid {{ oc_provision_vlan }}"
+  - lldp transmit-tlv port-desc sys-name
+
+# Interface config for monitoring nodes.
+switch_interface_config_monitoring:
+  - spanning-tree portfast
+  - switchport mode general
+  - "switchport general allowed vlan add {{ oc_provision_vlan }} untagged"
+  # Current expect-based configuration mechanism requires no line wrapping, so
+  # VLANs must be configured individually.
+  - "switchport general allowed vlan add {{ internal_vlan }} tagged"
+  - "switchport general pvid {{ oc_provision_vlan }}"
+  - lldp transmit-tlv port-desc sys-name
+
+# Interface config for Ceph storage nodes.
+switch_interface_config_ceph:
+  - spanning-tree portfast
+  - switchport mode general
+  - "switchport general allowed vlan add {{ oc_provision_vlan }} untagged"
+  # Current expect-based configuration mechanism requires no line wrapping, so
+  # VLANs must be configured individually.
+  - "switchport general allowed vlan add {{ internal_vlan }} tagged"
+  - "switchport general allowed vlan add {{ storage_vlan }} tagged"
+  - "switchport general allowed vlan add {{ storagemgmt_vlan }} tagged"
+  - "switchport general pvid {{ oc_provision_vlan }}"
+  - lldp transmit-tlv port-desc sys-name
+
+switch_interface_config_trunk_bmcs:
+  - "switchport mode trunk"
+  - "switchport trunk allowed vlan add {{ oc_oob_vlan }}"
+  - "switchport trunk allowed vlan add {{ wl_oob_vlan }}"
```

Comment:
Do these default to tagged? I'm not sure it would work having multiple untagged VLANs on a trunk.